apps/ui/src/views/SpaceUser/Votes.vue

```diff
@@ -0,0 +1,5 @@
+<script setup lang="ts"></script>
+
+<template>
+  <div></div>
+</template>
```

Comment:
No design for delegates, votes and proposals? right now it show empty
<img width="863" alt="Untitled 7" src="https://github.com/user-attachments/assets/cc7e196a-01d8-4010-bd75-c4d2c73d3246">

Reply:
Indeed. And UI is also not ready for these pages, afaik